filters/filter_17_TrackParam/metadata.json

```diff
@@ -13,11 +13,7 @@
   ],
   "platformsExcluded": [
     "ios",
-    "ext_safari",
-    "ext_android_cb",
```

Comment:
`ext_android_cb` does not support `removeparam`

Reply:
reverted